src/codewhisperer/service/transformationResultsViewProvider.ts

```diff
@@ -278,6 +278,7 @@ export class ProposedTransformationExplorer {
                 )
             } catch (e: any) {
                 // This allows the customer to retry the download
+                vscode.window.showErrorMessage('Transform by Q experienced an error when downloading the diff')
```

Comment:
Better if you can put these inline strings into constants file in case you reference them in multiple places.

Reply:
Agreed, will do so in next PR. There are more than a couple of places where this should be done